packages/client/vue.config.js

```diff
@@ -16,12 +16,13 @@ module.exports = {
     },
   },
   devServer: {
+    allowedHosts: process.env.VUE_ALLOWED_HOSTS || 'auto',
     client: {
       progress: false,
     },
     proxy: {
       '/api': {
-        target: 'http://localhost:3000',
+        target: process.env.VUE_APP_API_URL || 'http://localhost:3000',
```

Comment:
why was it necessary to make this an env var?

nit: name it differently, this use has nothing to do with Vue and `VUE_APP_*` environment variables have [special semantics](https://cli.vuejs.org/guide/mode-and-env.html#environment-variables) (namely that Vue CLI configures webpack to embed those environment variables in the client bundle)

Reply:
> why was it necessary to make this an env var?

Within the Docker Compose runtimes, the cross-container hostnames are expose according to the respective service names defined in `docker-compose.yml` rather than `localhost`, so (e.g.) the `frontend` container contacts the backend at `http://app:3000` rather than `http:localhost:3000`. I didn't want to break anyone's existing dev setup by assuming they'd be using Docker, so I opted to have this configurable via environment variable rather than hard-coding the container service name.

Regarding the nit- that's a good callout. I'll do something like `GOST_API_URL` instead.